spec/queries/search_results_query_spec.rb

```diff
@@ -7,10 +7,10 @@
     context 'given NO filters' do
       it 'returns all search results' do
         user = Fabricate(:user)
```

Comment:
Revisit this setup based on https://github.com/nimblehq/ic-rails-huey-toby/pull/51/files#r1266122433